src/common/composables/use-navigation/use-navigation.ts

```diff
@@ -36,6 +36,10 @@ export function useNavigation () {
     return router.push(ROUTE_NAMES.STUDENT.TOOLS_RESUMES)
   }
 
+  const navigateToStudentSelfKnowledgeCategories = () => {
+    return router.push(ROUTE_NAMES.STUDENT.SELFKNOWLEDGE_CATEGORIES)
```

Comment:
```suggestion
  const navigateToStudentSelfKnowledgeCategory = () => {
    return router.push(ROUTE_NAMES.STUDENT.SELFKNOWLEDGE_CATEGORY)
```

On est sur le détail d'une catégorie et pas une vue de l'ensemble des catégories